src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -3819,6 +3833,29 @@ public static IEnumerable<object[]> JsonWithValidTrailingCommasAndComments
             }
         }
 
+        [Theory]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF, (byte)'1' }, true, 1)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF, (byte)'1' }, true, 2)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF, (byte)'1' }, true, 3)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF, (byte)'1' }, false, 1)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF, (byte)'1' }, false, 2)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF, (byte)'1' }, false, 3)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF }, true, 1)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF }, true, 2)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF }, true, 3)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF }, false, 1)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF }, false, 2)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF }, false, 3)]
+        public static void TestBOMWithSingleJsonValueMultiSegment(byte[] utf8BomAndValue, bool isFinalBlock, int segmentSize)
```

Comment:
Move this to the file with the rest of the multi-segment tests:
https://github.com/dotnet/corefx/blob/42ecf717478504fd7cc072f3586c6edd1ba57daa/src/System.Text.Json/tests/Utf8JsonReaderTests.MultiSegment.cs#L13

Reply:
Looks like this feedback was never addressed.

`TestBOMWithSingleJsonValueMultiSegment` should be in `Utf8JsonReaderTests.MultiSegment.cs` and `TestBOMWithSingleJsonValue` should be in `Utf8JsonReaderTests.cs`

Currently, it's the other way around which caused some confusion for me.

Reply:
Fixed this in https://github.com/dotnet/corefx/pull/40349/commits/cf9ec7d663bb4ce526ad0ad209e80a684d0d35e5 as part of https://github.com/dotnet/corefx/pull/40349